src/cms/templates/pages/page_tree.html

```diff
@@ -117,6 +100,52 @@ <h2 class="heading font-normal mb-2">{% trans 'Pages' %}</h2>
         </tbody>
     </table>
 </div>
+<div class="table-footer">
+    <div class="flex flex-wrap" style="padding-top: 0.6em;">
+        <div class="flex-wrap relative w-1/2">
+            <form class="table-search relative flex flex-wrap h-full">
+                <div class="pl-3" style="padding-top: 0.4em;">
+                    {% trans 'Bulk actions' %}
+                </div>
+                <div class="pl-3">
+                    <select class="h-full pl-4 pr-4 rounded shadow" style="padding-top: 5px;" name="bulk_action" id="bulk_action">
+                        <option>{% trans 'Please select' %}</option>
+                        <option value="archive_pages">{% trans 'Archive pages' %}
+                        {% for lang in languages %}
+                        {% if lang != region.default_language %}
+                        <option value="{{ lang.code }}">{% trans 'Export XLIFF for translation to' %} {{ lang.translated_name }}</option>
+                        {% endif %}
+                        {% endfor %}
+                    </select>
```

Comment:
Actually the translation should be done according to the translation tree only. That means if German is translated to English and English to Arabic, the source for Arabic should be English when exporting to XLIFF. And I think we should only allow exactly this to prevent any misunderstandings.

Reply:
Ah sorry, I didn't catch that the source language is already determined by the language tree. Perfect!
Yes, I totally agree with you. I just thought maybe it would be good to have the source language also in the select option field to make sure there are absolutely no misunderstandings about which language is the source, e.g.
"Export XLIFF for translation from German to English", "Export XLIFF for translation from English to Arabic", etc... but I think that's optional, since the region managers should know their "translation directions"...

Reply:
Yes, I thought about this option but went for the shorter version. It is easy to change at any time later though. Just a minor adjustment in the template.